docs/common/decorators/secrets.md

```diff
@@ -0,0 +1,39 @@
+Secrets are pieces of senstivie information stored securely by Atomist. Secrets
+are used by [Rug Functions](/user-guide/rug/rug-functions.md) to provide access
+to secured systems, such as the GitHub API.
+
+Handlers that invoke Rug Functions that require secrets must use the
+`#!typescript @Secrets` decorator to declare to the Rug runtime that those
+secrets will be required during the execution of the handler's CommandPlan:
+
+```typescript
+...
```

Comment:
Are these ellipses needed?

I'd shy away from using ellipses in TypeScript code since it is a valid operator.